Gemfile.lock

```diff
@@ -249,3 +272,6 @@ DEPENDENCIES
   uglifier (= 2.7.2)
   unicorn (~> 4.9.0)
   webmock (= 1.21.0)
+
```

Comment:
It'll be fine, but once rebased make sure it's still the correct version for ruby 2.3.0.